packages/client/src/index.test.ts

```diff
@@ -58,7 +57,11 @@ describe('init client', () => {
       expect(1).toEqual(1);
```

Comment:
?

Reply:
same problem to `console.log`, the tests here need to be run twice, is dummy, will fix it someday

Reply:
looks like these `console.log` doesn't make sense in UTs. we need always expect something.